dts/bindings/usb/zephyr,hid-device.yaml

```diff
@@ -36,11 +36,11 @@ properties:
       The size of the longest input report that the HID device can generate.
       This property is used to determine the buffer length used for transfers.
 
-  in-polling-rate:
+  in-polling-period-us:
```

Comment:
oh yes, that is right. Can you please drop `_us`, there will be no other similar property and the description is clear about the unit.

Reply:
isn't it conventional in dt properties to specify the unit? am I making things up?

Reply:
I prefer to have the units in member names.

Reply:
Not sure it is conventional, but okay, convinced

Reply:
@jfischer-no  thanks, you are still blocking though :-)

Reply:
Yes, to maintain my bad reputation. (I just wait for @MarekPieta reaction)

Reply:
Accepted. We will just need to apply the rename to our application (since USB next is still experimental it should not be too problematic).

Reply:
Right, for the upstream project, this has been originally introduced in this release so it should not be a problem to change it at this stage.